pkg/controller/quayecosystem/configuration/reconcile.go

```diff
@@ -520,6 +636,91 @@ func (r *ReconcileQuayEcosystemConfiguration) quayDeployment(meta metav1.ObjectM
 
 }
 
+func (r *ReconcileQuayEcosystemConfiguration) quayConfigDeployment(meta metav1.ObjectMeta) error {
+
+	meta.Name = resources.GetQuayConfigResourcesName(r.quayEcosystem)
+	resources.BuildQuayConfigResourceLabels(meta.Labels)
+
+	quayDeploymentPodSpec := corev1.PodSpec{
+		Containers: []corev1.Container{{
+			Image: r.quayEcosystem.Spec.Quay.Image,
+			Name:  meta.Name,
+			Env: []corev1.EnvVar{
+				{
+					Name:  constants.QuayEntryName,
+					Value: constants.QuayEntryConfigValue,
+				},
+				{
+					Name:  constants.QuayConfigPasswordName,
+					Value: "quay",
+				},
+			},
+			Ports: []corev1.ContainerPort{{
+				ContainerPort: 8080,
+				Name:          "http",
+			}, {
+				ContainerPort: 8080,
+				Name:          "https",
```

Comment:
Similarly here, 8080 instead of 8443

Reply:
@thomasmckay updated as per pervious comment